DEVELOPMENT.md

```diff
@@ -45,6 +45,86 @@ At this point, you may find it useful to return to these `Tekton Pipeline` docs:
 -   [Tekton Pipeline "Hello World" tutorial](https://github.com/tektoncd/pipeline/blob/master/docs/tutorial.md) -
     Define `Tasks`, `Pipelines`, and `PipelineResources`, see what happens when
     they are run
+    
+## Environment Setup
+
+You must install these tools:
+
+1.  [`git`](https://help.github.com/articles/set-up-git/): For source control
+
+1.  [`go`](https://golang.org/doc/install): The language Tekton Pipelines is
```

Comment:
Could you specify which version of golang is needed to be installed? If you install an old version. I believe that KO throws a very unhelpful error.

Reply:
Also, I think your need to edit your pr to include `/kind documentation`.

Reply:
just a side note that @AlexReiff was moving the existing docs around - adding the version is a good idea but i think it is np if he doesn't want to be the one to add it

Reply:
I debated this. I decided against this because it's more an issue with installing Ko than installing Tekton.